libs/ui/src/theme/global/globalStyles.tsx

```diff
@@ -0,0 +1,24 @@
+import { normalize } from 'polished';
+import { createGlobalStyle } from 'styled-components';
+import { Theme } from '..';
+import { Font } from '.';
+
+export const GlobalStyles = createGlobalStyle`
+  ${normalize()}
+
+  * {
+    &,
+    &::before,
+    &::after {
+      box-sizing: border-box;
+    }
+  }
+  html {
+    font-size: 10px;
```

Comment:
Great call -- has design also aligned with this? Mainly I'm wondering if the spacing values they're using in Figma are still in a 4-base px system. Not that it matters much, but I've found it tricky to have a rem = 10px when working with a 4-base px system in Design.